addons/hi-res-thumbnails/addon.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Higher resolution thumbnails",
+  "description": "Replaces thumbnails across the website with higher resolution images. Useful on retina displays or browsing at >100% zoom.",
+  "info": [
+    {
+      "type": "warning",
+      "text": "Higher resolution images require larger filesizes. Take care not to accidentally incur data expenses if your network plan is limited.",
+      "id": "imagefilesize"
+    }
+  ],
+  "settings": [
+    {
+      "name": "Dimension Multiplier (%)",
+      "id": "multiplier",
+      "type": "integer",
+      "min": 100,
+      "max": 300,
+      "default": 200
```

Comment:
This is a percentage rather than a decimal (e.g. 2.00x) because I'm not exactly sure decimal settings exist, and because integers are generally nicer for users to play with (citation needed).

It'd be nice to have something like `"step": 10` here (for nicer non-typing controls), but I don't *think* there exists support for that in the current settings viewer?

Reply:
I agree that there should be a `step` option - it would also be helpful for addons that already have percentage settings, such as block transparency.

Reply:
#3860